### PR TITLE
VueUI 2 Part 2 - Reporting tool

### PR DESCRIPTION
**Pull request by Karolis2011**
This is a tool that makes vueui reports instead of CI server doing same thing. 
image

### CODE REVIEW

**Comment by Karolis2011** [comment]
To be clear VueUI 2.0 is currently going to be made up of multiple partially related PRs. This PR is part of VueUI 2.0 a.k.a. #9526

**Comment by Karolis2011** [comment]
!review

File: vueui/tests/manifest.json [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+  "size": [580, 700],
+  "data": {
+    "state": {
+      "manifest": {
+        "Command": [
+          {
+            "name": "Xiao Zemin",
+            "rank": "Research Director",
+            "active": "Active",
+            "head": 0
```
Comment (NonQueueingMatt) [inline vueui/tests/manifest.json:11]: Is this meant to be here?

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Geevies — APPROVED** [review]

**Review by Arrow768 — CHANGES_REQUESTED** [review]
There is no description to explain how to use this tool.

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**